packages/core/package.json

```diff
@@ -13,12 +13,17 @@
     "@stencil/core": "0.0.5-23",
     "@stencil/dev-server": "latest",
     "@stencil/utils": "latest",
+    "@types/jest": "^20.0.8",
+    "jest": "^21.1.0",
+    "memory-fs": "^0.4.1",
```

Comment:
I forgot about this one before. Had to do the same for my `stencil-toh` project that I originally was testing all of this in. Should we make `memory-fs` a peer-dep? or should we just document the fact that it needs to be installed when ppl do unit tests?

I prefer the former, but I know you are also trying to keep the deps to a minimum.